docs/src/main/sphinx/functions/url.rst

```diff
@@ -40,7 +40,26 @@ such as ``:`` or ``?``.
 
 .. function:: url_extract_protocol(url) -> varchar
 
-    Returns the protocol from ``url``.
+    Returns the protocol from ``url``:
+    ::
```

Comment:
remove this and just have double :: above

Reply:
removed

Reply:
You misunderstood. Sorry ... use this:

```
Returns the protocol from ``url``::
```